src/saveload/afterload.cpp

```diff
@@ -1548,6 +1548,16 @@ bool AfterLoadGame()
 		}
 	}
 
+	if (IsSavegameVersionBefore(SLV_PROTECT_PLACED_HOUSES)) {
+		for (auto t : Map::Iterate()) {
+			if (IsTileType(t, MP_HOUSE)) {
+				/* House protected flag is stored in the map, instead of checked against GRF spec. */
```

Comment:
I do not know what this comment is trying to say to me. It most likely make sense to you with history etc, but to me this is just .. gibberish :D

Maybe you mean something like:

`Default the house protected flag to the flag in the GRF`, or something?

Mostly, I do not know what `GRF spec` has to do with this line of code, and what you are checking against. I just can't make sense out of it :)